content/fr/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,344 @@
+kubectl rollout status -w deployment/frontend                    # Écoute (Watch) le status du rolling update du déploiement "frontend" jusqu'à ce qu'il se termine
+
+# déprécié depuis la version 1.11
+kubectl rolling-update frontend-v1 -f frontend-v2.json           # (déprécié) Rolling update des pods de frontend-v1
+kubectl rolling-update frontend-v1 frontend-v2 --image=image:v2  # (déprécié) Modifie le nom de la ressource et met à jour l'image
+kubectl rolling-update frontend --image=image:v2                 # (déprécié) Met à jour l'image du pod du déploiement frontend
+kubectl rolling-update frontend-v1 frontend-v2 --rollback        # (déprécié) Annule (rollback) le rollout en cours
+
+cat pod.json | kubectl replace -f -                              # Remplace un pod, en utilisant un JSON passé en entrée standard
+
+# Remplacement forcé (Force replace), supprime puis re-crée la ressource. Provoque une interruption de service.
+kubectl replace --force -f ./pod.json
+
+# Crée un service pour un nginx repliqué, qui rend le service sur le port 80 et se connecte aux containers sur le port 8000
+kubectl expose rc nginx --port=80 --target-port=8000
+
+# Modifie la version (tag) de l'image du container unique du pod à v4
+kubectl get pod mypod -o yaml | sed 's/\(image: myimage\):.*$/\1:v4/' | kubectl replace -f -
+
+kubectl label pods my-pod new-label=awesome                      # Ajoute un Label
+kubectl annotate pods my-pod icon-url=http://goo.gl/XXBTWq       # Ajoute une annotation
+kubectl autoscale deployment foo --min=2 --max=10                # Mise à l'échelle automatique (Auto scale) d'un déploiement "foo"
+```
+
+## Mise à jour partielle (Patch) de ressources
+
+```bash
+kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}' # Met à jour partiellement un noeud
+
+# Met à jour l'image d'un container ; spec.containers[*].name est requis car c'est une clé du merge
+kubectl patch pod valid-pod -p '{"spec":{"containers":[{"name":"kubernetes-serve-hostname","image":"new image"}]}}'
+
+# Met à jour l'image d'un container en utilisant un patch json avec tableaux indexés
+kubectl patch pod valid-pod --type='json' -p='[{"op": "replace", "path": "/spec/containers/0/image", "value":"new image"}]'
+
+# Désactive la livenessProbe d'un déploiement en utilisant un patch json avec tableaux indexés
+kubectl patch deployment valid-deployment  --type json   -p='[{"op": "remove", "path": "/spec/template/spec/containers/0/livenessProbe"}]'
+
+# Ajoute un nouvel élément à un tableau indexé 
+kubectl patch sa default --type='json' -p='[{"op": "add", "path": "/secrets/1", "value": {"name": "whatever" } }]'
+```
+
+## Édition de ressources
+Ceci édite n'importe quelle ressource de l'API dans un éditeur.
+
+```bash
+kubectl edit svc/docker-registry                      # Édite le service nommé docker-registry
+KUBE_EDITOR="nano" kubectl edit svc/docker-registry   # Utilise un autre éditeur
+```
+
+## Mise à l'échelle (Scale) de ressources
```

Comment:
as we discussed on slack we all agree to use this formula for first occurence 'french ( english )',
but what about sections titles ? personnaly i'm not sure.